SampleGetApi/ApiHelper/ApiCommonHelper.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace SampleGetApi.ApiHelper
+{
```

Comment:
Empty class?

Reply:
Its a Draft class which can be used to Modularise the API Calls. Added just to show case the framework Structure. Will add a comment explaining this part.